docs/_spec/01-lexical-syntax.md

```diff
@@ -169,17 +169,16 @@ A newline in a Scala source text is treated as the special token “nl” if the
 
 The tokens that can terminate a statement are: literals, identifiers and the following delimiters and reserved words:
 
-```scala
+```
 this    null    true    false    return    type    <xml-start>
 _       )       ]       }
 ```
 
 The tokens that can begin a statement are all Scala tokens _except_ the following delimiters and reserved words:
 
-```scala
-catch    else    extends    finally    forSome    match
-with    yield    ,    .    ;    :    =    =>    <-    <:    <%
->:    #    [    )    ]    }
+```
+catch    else    extends    finally    forSome    match    with    yield
+,    .    ;    :    =    =>    <-    <:    <%    >:    #    [    )    ]    }
```

Comment:
```suggestion
,    .    ;    :    =    =>    <-    <:    <%    >:    #    )    ]    }
```
looks like this can go due to polymorphic lambdas now being added